packages/groq-builder/src/commands/project.test.ts

```diff
@@ -650,8 +650,8 @@ describe("project (object projections)", () => {
 
       await expect(() => executeBuilder(qParser, { datalake: invalidData }))
         .rejects.toThrowErrorMatchingInlineSnapshot(`
-        "1 Parsing Error:
-        result[5].price: Expected number, received string"
+        [ValidationErrors: 1 Parsing Error:
```

Comment:
It appears that the snapshot changed here to include the ErrorType. At this point it is unclear why. Will investigate.